PyPI_README.md

```diff
@@ -4,23 +4,290 @@ Welcome to the SaladCloud Transcription SDK documentation. This guide will help
 
 ## Versions<a id="versions"></a>
 
-- API version: `0.9.0-alpha.1`
-- SDK version: `0.9.0-alpha.1`
+- API version: `1.0.0a1`
+- SDK version: `1.0.0a1`
```

Comment:
We're using SemVer 2.0 as the standard for describing our versions. I understand PyPI has a different version syntax, but I think in docs the SemVer is still valid. Should we keep this as a SemVer?